vm/src/vm/errors/hint_errors.rs

```diff
@@ -192,6 +192,10 @@ pub enum HintError {
     ExcessBalanceKeyError(Box<str>),
     #[error("excess_balance_func: Failed to calculate {0}")]
     ExcessBalanceCalculationFailed(Box<str>),
+    #[error("generate_nibbles fail: tried to read from an empty list of nibbles")]
+    EmptyNibbles,
+    #[error("bls_split failed: BigInt could not be split")]
+    BlsSplitFail,
```

Comment:
I think this variant is not used anywhere. If that's the case, please remove it.

Reply:
Aah yes, this is for a KZG hint to be implemented in a different PR. On it!

Reply:
Resolved by https://github.com/lambdaclass/cairo-vm/pull/1829#:~:text=fix%3A%20remove%20unused%20error%20variant